### ISSUE TITLE
Sustain Bypass Note Hold

### ISSUE THREAD

**Issue by baconpaul** [surge-synthesizer/surge#6506]
Inbound request from a user with a device that has a sustain pedal but wants to use it *just* as a modulator to have the sustain modulator present an option to *not* do voice holds but instead just send information into the mod matrix.

Ponder ponder. But tossing into 1.1 also in case ponder becomes ship. May bounce out.

**Comment by Andreya-Autumn** [comment]
Not sure why one wouldn't just put the pedal on another CC than 64, then use MIDI learn?

Maybe to make patches that other users could play without changing their pedal away from CC64?

If so, that might be cool. But maybe a bit unexpected since 100% of patches so far have the same sustain pedal behavior?

**Comment by mkruselj** [comment]
Yeah actually I would just remap to CC2 or CC11 and use that instead.

**Comment by Andreya-Autumn** [comment]
Are there many controllers out there without the ability to remap sustain off of 64? 
That could be another possible grounds to incorporate this FR if so. 
I'm not knowledgeable enough about the state of MIDI controllers rn to know this. My impression is that most serious master keyboards can do this, but how common is it in lower-grade options?

**Comment by mkruselj** [comment]
A lot of them can, yeah. Not the cheapest ones usually, but anything that's halfways decent can remap stuff. Plus you can also remap things in your DAW usually.

**Comment by baconpaul** [comment]
So the nature of this request was: Having to reconfigure your device when changing a patch to learn the patch author intent is a bit painful. (This came from a linnstrument user). But I also thought of all the thigns you said. let me let this one brew in our heads for a bit since there’s an ergonomics in the standalone of having this feature which is useful, but also all the other things you said are indeed 100% correct

**Comment by mkruselj** [comment]
I'm not sure standalone is our greatest and most important use case, so...

**Comment by mkruselj** [comment]
This is quite easy to do so I'm pulling it into 1.4...
